db/migrate/20210114143800_nilify_empty_proxy_credentials.rb

```diff
@@ -1,19 +1,11 @@
 class NilifyEmptyProxyCredentials < ActiveRecord::Migration[6.0]
   def up
-    if User.unscoped.find_by_login(User::ANONYMOUS_ADMIN).present?
-      User.as_anonymous_admin do
```

Comment:
Was this here to disable auditing or something? This is fairly recent (d60d1cb169ac83a8aed6ba7256c1b731dce07350) so maybe @ares remembers.

Reply:
I'm not sure, possibly an attempt to bypass taxonomy or authorization scope. `update_all` triggers sql `UPDATE` query directly without initializing AR objects so it wouldn't have been audited in any case.